### ISSUE TITLE
Typedef `call-signature` is not working on arrow functions

### ISSUE THREAD

**Issue by sergiubologa** [palantir/tslint#1015]
See the following example:

```javascript
export class TsLint {
    function helloWorld() {
        return 'Hello World';
    };

    arrowHelloWorld = () => {
        return helloWorld();
    };
}
```

In `tslint.json` file I have the following rule configured:
```javascript
"typedef": [
      true,
      "call-signature",
      "parameter",
      "arrow-parameter",
      "property-declaration",
      "variable-declaration",
      "member-variable-declaration"
    ],
```

When running tslint I get an error only for the `helloWorld` function and not for the arrow function `arrowHelloWorld ` also:
```console
[08:59:38] Starting 'tslint'...
[08:59:38] [gulp-tslint] error (typedef) tslintIssue.ts[2, 25]: expected call-signature: 'helloWorld' to have a typedef
[08:59:38] 'tslint' errored after 175 ms
[08:59:38] Error in plugin 'gulp-tslint'
Message:
    Failed to lint: tslintIssue.ts[2, 25]: expected call-signature: 'helloWorld' to have a typedef.
```

Is this an issue in tslint?

**Comment by JKillian** [comment]
Thanks for the report @sergiubologa, seems like a bug to me!
